tests/execution_test.go

```diff
@@ -209,6 +217,26 @@ func TestListWorkflowExecutions(t *testing.T) {
 	assert.Equal(t, len(resp.Executions), 4)
 }
 
+func TestListWorkflowExecutionsWithTags(t *testing.T) {
+	truncateAllTablesForTestingOnly()
+	populateWorkflowExecutionsForTestingOnly()
+
+	ctx := context.Background()
+	client, conn := GetTestAdminServiceClient()
+	defer conn.Close()
+
+	resp, err := client.ListExecutions(ctx, &admin.ResourceListRequest{
+		Id: &admin.NamedEntityIdentifier{
+			Project: "project1",
+			Domain:  "domain1",
+		},
+		Limit:   5,
+		Filters: "value_in(admin_tag.name, hello)",
+	})
+	assert.Nil(t, err)
+	assert.Equal(t, len(resp.Executions), 2)
+}
```

Comment:
given that's not too difficult to write integration tests, can we validate what happens when duplicate labels are used as part of creating an execution?

Reply:
It's hard to do that here since we must register a workflow first. In these integration tests, we didn't really create any real workflow in the database.